### DISCUSSION TITLE
We need to improve the code execution process.

### DISCUSSION THREAD

**Discussion by crimson206** [microsoft/autogen#1773] [Ideas]
Please read the issue #1772.

When we get an error message from the execution of a code written by a Chatbot,
we don't simply try to resolve the error, but can make much different approaches.

With the simple setup - userproxy <-> assistant,
we'd better make some recommended prompts, with a default one who is the most stable.

**Comment by ekzhu** [comment]
Yes. Prompting is big issue when it comes to windows. The default system prompt for AssistantAgent may not work well for windows cmd. If you can come up with some good prompts we can add to our example gallery. 

Also please take a look at autogen.coding for code exexutors. You can create a new one just for windows cmd.
